CSS/hintwork-style.css

```diff
@@ -0,0 +1,217 @@
+.work-hint-modal-container {
+  position: fixed;
+  top: 5%;
+  right: 1%;
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  opacity: 0;
+  pointer-events: none;
+  transition: opacity 0.5s ease-in-out;
+  z-index: 10;
+  background-color: #fff;
+  padding-top: 30px;
+  padding-left: 20px;
+  padding-right: 20px;
+}
+
+.work-hint-modal-container.active {
+  opacity: 1;
+  pointer-events: auto;
+}
+
+.work-hint-modal {
+  width: 100%;
+  margin: 0 5px;
+  background-color: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.5);
+  position: relative;
+}
+
+.work-hint-modal h2 {
+  display: inline;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 28px;
+  line-height: 44px;
+  color: #172b4d;
+  letter-spacing: 0.8px;
+  margin: 16px;
+}
+
+.work-hint-modal button {
+  border: none;
+  cursor: pointer;
+  font-size: 30px;
+  font-weight: bold;
+  position: absolute;
+  top: 10px;
+  right: 20px;
+  opacity: 0.3;
+  background: none;
+}
+
+.work-hint-modal button:hover {
+  opacity: 1;
+}
+
+.work-hint-modal ul {
+  display: flex;
+  flex-direction: row;
+  width: 100%;
+  margin: 16px;
+}
+
+.work-hint-modal li {
+  font-family: 'Poppins', sans-serif;
+  background: #fff;
+  border: 1px solid #979493;
+  padding: 10px 12px;
+  font-weight: bold;
+  font-size: 14px;
+  line-height: 24px;
+  color: #3c3a39;
+  margin-right: 16px;
+  list-style: none;
+}
+
+.work-hint-placeholder {
+  width: auto;
+  height: auto;
+  margin: 0 16px;
+}
+
+.work-hint-modal p {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  color: #344563;
+  opacity: 0.8;
+  margin: 0 16px;
+}
+
+.d-links {
+  margin: 0 16px;
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-start;
+  width: 90%;
+  padding-bottom: 20px;
+}
```

Comment:
These links, at the moment, are glued together in mobile view.
Your design:
![image](https://user-images.githubusercontent.com/83596135/162236919-86c3f9ea-4567-4349-afc6-1d04c7dbb25c.png)
Figma design:
![image](https://user-images.githubusercontent.com/83596135/162236982-02b1f39e-8eb1-4bdd-8009-0ae1ea67e418.png)

- [ ] In order to respect Figma design, please take measures to display the buttons accordingly.
- [ ] As well, kindly I ask, to reduce the width of the buttons in order to match figma design.